skins/laika/src/components/pages/membership_form/Address.vue

```diff
@@ -116,10 +117,23 @@ export default Vue.extend( {
 				}, ( {} as AddressValidity ) );
 			},
 		},
+		addressTypeFromName: {
+			get: function () {
+				return addressTypeFromName;
+			},
+		},
 		...mapGetters( NS_MEMBERSHIP_ADDRESS, [
 			'addressType',
 		] ),
 	},
+	mounted() {
+		const initialFormValues = this.$props.initialFormValues;
+		if ( initialFormValues !== undefined ) {
+			Object.keys( this.$data.formData ).forEach( ( fieldName: string ) => {
+				this.formData[ fieldName ].value = initialFormValues[ fieldName ];
+			} );
+		}
+	},
```

Comment:
Writing a mounted function was more convenient than going through the entire `formData` object and assigning the appropriate `initialFormValue`. Plus I'd always have to check if the object is undefined, so `formData` becomes even hareder to read. 
Since we need to populate the form only in case of the donor coming from the donation confirmation page I thought this is easier and more isolated.
`AddressType` has only one data attribute so it's not hard to read. Either way is fine there.